### PR TITLE
Remove WiringPi dependency

### PR DESCRIPTION
**Pull request by abouillot**
Remove wiringpi dependency by directly using /sys/class/gpio commands ato toggle PIN 22 - GPIO 25

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -96,14 +96,6 @@ INSTALL_DIR="/opt/ttn-gateway"
 if [ ! -d "$INSTALL_DIR" ]; then mkdir $INSTALL_DIR; fi
 pushd $INSTALL_DIR
 
-# Remove WiringPi built from source (older installer versions)
```
Comment (gonzalocasas) [inline install.sh:99]: This removal part is still needed because we need to keep the already installed gateways clean

File: install.sh [hunk 2]
```diff
@@ -136,7 +128,6 @@ popd
 
 # Install dependencies
```
Comment (gonzalocasas) [inline install.sh]: Please remove the comment and `echo` now that no dependencies are needed anymore

File: start.sh [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
 
 # Reset iC880a PIN
-gpio -1 mode 22 out
-gpio -1 write 22 0
-sleep 0.1
-gpio -1 write 22 1
-sleep 0.1
-gpio -1 write 22 0
-sleep 0.1
+pin=25
```
Comment (gonzalocasas) [inline start.sh]: could you please use all upper case for the variable and a more descriptive name, like `SX1301_RESET_BCM_PIN` (25 is 22 in BCM2835 denomination, right?)

File: start.sh [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
+echo "$pin"  > /sys/class/gpio/export 
+echo "out" > /sys/class/gpio/gpio$pin/direction 
+echo "0"   > /sys/class/gpio/gpio$pin/value 
+sleep 0.1  
+echo "1"   > /sys/class/gpio/gpio$pin/value 
+sleep 0.1  
+echo "0"   > /sys/class/gpio/gpio$pin/value
+echo "$pin"  > /sys/class/gpio/unexport
```
Comment (gonzalocasas) [inline start.sh]: please add a `sleep 0.1` between the last `0` and the `unexport` echos

**Comment by gonzalocasas** [comment]
Really cool stuff! Thanks @abouillot for all the PRs! I added some feedback, once addressed, I will merge! :+1:

**Comment by abouillot** [comment]
Comments are now integrated

**Pull request merged**